### PR TITLE
ZEPPELIN-360 Remove "Copyright 2014, NFLabs inc." from LICENSE

### PR DESCRIPTION
**Pull request by Leemoonsoo**
This PR addresses https://issues.apache.org/jira/browse/ZEPPELIN-360
Removing "Copyright 2014, NFLabs inc." from LICENSE.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014, NFLabs inc.
+   Copyright [yyyy] [name of copyright owner]
```
Comment (jongyoul) [inline LICENSE]: @Leemoonsoo Is it enough? Will it be filled automatically?

Comment (Leemoonsoo) [inline LICENSE]: I checked some TLPs to see how they do. And looks like it's enough. I'm also curious if it is enough (and why).

Comment (jongyoul) [inline LICENSE]: Thanks. I'm also curious :)

**Comment by felixcheung** [comment]
looks good? :)

File: LICENSE (hunk 1 above)
Comment (Leemoonsoo) [inline LICENSE]: Can one of our mentors clarify it?

Comment (felixcheung) [inline LICENSE]: This section is talking about "How to apply the Apache License to your work."
 
```
  APPENDIX: How to apply the Apache License to your work.        To apply the Apache License to your work, attach the following       boilerplate notice, with the fields enclosed by brackets "[]"       replaced with your own identifying information. (Don't include       the brackets!)  The text should be enclosed in the appropriate       comment syntax for the file format. We also recommend that a       file or class name and description of purpose be included on the       same "printed page" as the copyright notice for easier          identification within third-party archives. ```
 
So I think [yyyy] [name of copyright owner] is indeed correct.

Comment (Leemoonsoo) [inline LICENSE]: Thanks, I'm merging it.

**Pull request closed without merging**